### ISSUE TITLE
Unable to run docker image 

### ISSUE THREAD

**Issue by BryceCicada** [MISS3D/s2p#201]
Steps to reproduce on current master (7b12beb):

- `docker build --no-cache . -t s2p`
- `docker run --rm -it s2p python s2p.py --help`
- Witness: 
  ```
  Traceback (most recent call last):
  File "s2p.py", line 33, in <module>
    import rasterio
  ImportError: No module named rasterio
  ```
Possibly from https://github.com/MISS3D/s2p/commit/d81966da37451e8b824aa3d5d5d7dd83b11d80d2

**Comment by BryceCicada** [comment]
Currently working around with small change to Dockerfile that reverts to an older version of s2p:
```
diff --git a/Dockerfile b/Dockerfile
index d381520..3917c5e 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -33,6 +33,6 @@ RUN apt-get update && apt-get install -y \
 
 # Install s2p from MISS3D/s2p
 RUN git clone https://github.com/MISS3D/s2p.git --recursive
-RUN cd s2p && make all
+RUN cd s2p && git checkout cfec24e && make all
 
 WORKDIR /s2p
```

**Comment by carlodef** [comment]
Should be fixed by pull request #205
